snapshots/who/2024-03-24/self_inflicted_injuries.csv.dvc

```diff
@@ -0,0 +1,38 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: WHO Mortality Database
+    description: |-
+      The WHO mortality database is a collection death registration data including cause-of-death information from member states.
+
+      Where they are collected, death registration data are the best source of information on key health indicators, such as life expectancy, and death registration data with cause-of-death information are the best source of information on mortality by cause, such as maternal mortality and suicide mortality.
+
+      WHO requests from all countries annual data by age, sex, and complete ICD code (e.g., 4-digit code if the 10th revision of ICD was used). Countries have reported deaths by cause of death, year, sex, and age for inclusion in the WHO Mortality Database since 1950.
+
+      Data are included only for countries reporting data properly coded according to the International Classification of Diseases (ICD). Today the database is maintained by the WHO Division of Data, Analytics and Delivery for Impact (DDI) and contains data from over 120 countries and areas. Data reported by member states and selected areas are displayed in this portal’s interactive visualizations if the data are reported to the WHO mortality database in the requested format and at least 65% of deaths were recorded in each country and year.
+    date_published: "2022-09-01"
+    title_snapshot: WHO Mortality Database - Self-inflicted Injuries
+    description_snapshot: |-
+      This dataset focuses specifically on mortality from self-inflicted injuries. 
+
+    # Citation
+    producer: WHO
```

Comment:
Maybe just for consistency with the other WHO MDB dataset, this could be 'WHO Mortality Database'. Also, the WHO has a bunch of similar-sounding variables, but with different methods, so I guess it helps to distinguish this a little.